### PR TITLE
add new column in L1 lgd_brand

### PR DESCRIPTION
**Pull request by fikrihaykal41**
column synchronization between snapshot and L1

### CODE REVIEW

File: dags/source/sql/dwh/L1/visibility/lgd_brand.sql [hunk 1]
```diff
@@ -1,15 +1,18 @@
 SELECT
   REPLACE(JSON_EXTRACT(data, '$.brandId'), '"', '') AS brand_id,
+  REPLACE(JSON_EXTRACT(data, '$.companyId'), '"', '') AS company_id,
   REPLACE(JSON_EXTRACT(data, '$.brandName'), '"', '') AS brand_name,
   JSON_EXTRACT_STRING_ARRAY(data, '$.categoryIds') AS category_ids,
+  CAST(REPLACE(JSON_EXTRACT(data, '$.isActive'), '"', '') AS BOOL) AS is_active,
   CAST(REPLACE(JSON_EXTRACT(data, '$.isDeleted'), '"', '') AS BOOL) AS is_deleted,
-  REPLACE(JSON_EXTRACT(data, '$.createdBy'), '"', '') AS created_by,
   CAST(REPLACE(JSON_EXTRACT(data, '$.createdAt'), '"', '') AS TIMESTAMP) AS created_at,
-  REPLACE(JSON_EXTRACT(data, '$.modifiedBy'), '"', '') AS modified_by,
```
Comment (toileto) [inline dags/source/sql/dwh/L1/visibility/lgd_brand.sql:8]: Kok `modifiedBy` nya dihapus?

Comment (fikrihaykal41) [reply dags/source/sql/dwh/L1/visibility/lgd_brand.sql:8]: eeee kayanya ini mas ada di bawahnya
image

**Review by toileto — APPROVED** [review]
Approved

**Review by ameliaksalwa — APPROVED** [review]
approved yey

**Pull request merged**